R/tm_g_bivariate.R

```diff
@@ -371,26 +370,33 @@ srv_g_bivariate <- function(id,
       selectors
     })
 
-    merged_data <- teal.transform::data_merge_srv(
+    anl_merged_input <- teal.transform::merge_expression_srv(
```

Comment:
### Explanation comment
merge module does not return `$data` anymore so we can't call it merged_data